nilearn/reporting/data/html/glm_report.html

```diff
@@ -161,21 +163,24 @@ <h2>Contrasts</h2>
     <div class="d-flex flex-column">
       {{if contrasts_dict}}
       <!--  -->
-      {{py: contrasts = contrasts_dict.items()}}
+      {{py: runs = contrasts_dict.items()}} {{for i_run, all_contrasts in runs}}
```

Comment:
this will plot the contrast for each contrast for each run, so may lead to having a lot of times the same plots but this allows users check the contrast matrix for each run

this messes up the UX / UI though as it is not clear which plot corresponds to which run: to fix in a follow up PR

https://output.circle-artifacts.com/output/job/886cf2ad-35da-4a5b-aaad-538834234dc7/artifacts/0/dev/auto_examples/04_glm_first_level/plot_two_runs_model.html#generating-a-report